ecommerce/baskets/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+from customers.models import Customer
+from products.models import Product
+from baskets import enums
+from django.utils.translation import gettext_lazy as _
+
+from core.models import BaseAbstractModel
+
+
+class Basket(BaseAbstractModel):
+    customer = models.ForeignKey(Customer, verbose_name=_("Customer"), null=True, on_delete=models.SET_NULL)
```

Comment:
blank=True parametresini de eklersek, formlardan blank olarak gelmesine de izin vermis oluruz. basket'i olan customer'in silinmesine izin vermezsek iyi olur.